.eslintrc.json

```diff
@@ -5,9 +5,10 @@
     "plugin:prettier/recommended",
     "plugin:import/errors",
     "plugin:import/warnings",
-    "plugin:import/typescript"
+    "plugin:import/typescript",
+    "plugin:testcafe/recommended"
```

Comment:
You could also use https://github.com/testcafe-community/eslint-plugin-testcafe-community (the latest release is from current year), but I don't know does it make any difference.

Reply:
hmm seems to have the same rules but also some additional stuff

Reply:
"plugin:testcafe/recommended" is the one mentioned in the testcafe docs